decode/clarity.ts

```diff
@@ -55,6 +58,8 @@ export function decode(input: string): DecodedPayload {
                 let metric = data.decode(entry) as MetricEvent;
                 // It's not possible to accurately include the byte count of the payload within the same payload
                 // So, we increment the bytes from the incoming payload at decode time.
+                // Alos, initialize TotalBytes if it doesn't exist. For the first payload, this value can be null.
```

Comment:
>Alos [](start = 19, length = 4)

also

Reply:
Thanks

---
In reply to: [347527567](https://github.com/microsoft/clarity-js/pull/160#discussion_r347527567) [](ancestors = 347527567)